ChangeLog.md

```diff
@@ -1,5 +1,9 @@
 # What's new in the WSL Management PowerShell module
 
+## Version 2.0.1 (2023-09-26)
+
+- Added support for retrieving WSL distribution listing from online sources via [`Get-WslDistributionOnline`][] cmdlet.
```

Comment:
Please use a maximum line width of 100 characters in Markdown files.

Reply:
Updated.